src/InputAndLabel.jsx

```diff
@@ -0,0 +1,16 @@
+import {useRef, useEffect} from "react";
+
+export default function InputAndLabel({ id, children, focus, ...inputProps}) {
```

Comment:
the spread operator here gathers all props that are not `id`, `children`, or `focus`. We then spread those props onto the `input` element.